addons/mrp/views/mrp_workorder_views.xml

```diff
@@ -363,6 +363,7 @@
                 decoration-warning="state == 'cancel' or (json_popover and 'text-warning' in json_popover)"
                 color="production_id"
                 display_unavailability="1"
+                progress_bar="workcenter_id"
```

Comment:
For some reason there are 2 gantt views for mrp.workorder. Do you need to add it to both or only this one?

Reply:
Ah nevermind about this one. Just looked at the enterprise PR. I will finish reviewing the enterprise PR tomorrow.